vuepress/docs/next/docs/getting-started/ent-api.md

```diff
@@ -50,21 +50,40 @@ Common operations associated with API claims are detailed below.
    - `bundle`: The external bundle URL
    - `service-name`: The name of a microservice in the external bundle. If `service-name` is not set, `api add-ext` will initiate an interactive mode where the user can select from available bundles and microservices.
 
-## API Claim YAML Descriptor Mapping
+- `api add-ext` requirements:
+   - Connection to an Entando instance
+   - The cluster contains the service specified in the API claim
+   - The external service has already been deployed
 
-When the YAML descriptor of a micro frontend is generated (i.e. via the `pack` command), the API claim mapping is similar to the following:
+- Executing `api add-ext` without flags:
+   - Returns the current instance
+   - Displays the instance's available services for interactive selection
 
-``` yaml
-apiClaims:
-	- name: int-api
-		type: internal
-		serviceName: int-ms
-	- name: ext-api
-		type: external
-		bundle: registry.hub.docker.com/cecchisandrone/entando-simple-bundle
-		serviceName: ext-ms
+## Microservice URL Retrieval
+
+To render an MFE installed from a bundle, Entando injects a JSON object containing configuration details inside the "config" attribute of a custom HTML element, e.g. `<my-mfe config="{ ... }" />`.
+
+**Installed Bundles**
+
+To retrieve the URL of an MS declared through an API claim, add `systemParams.api[claimName].url` to the MFE JavaScript code so it can access that element of the config object. The `claimName` was chosen by the user when the API claim was defined.
+
+Refer to the Entando demo bundles for alternative retrieval methods. These techniques are more robust and consider cases where the parent object is not defined.
```

Comment:
these techniques>> "These" techniques are referring to the alternative retrieval method ones, right? just checking

Reply:
yes, but i'll combine those 2 sentences to clarify